src/python/gudhi/simplex_tree.pyx

```diff
@@ -812,16 +812,19 @@ cdef class SimplexTree:
 
     def __eq__(self, other:SimplexTree):
         """Test for structural equality
+
         :returns: True if the 2 simplex trees are equal, False otherwise.
         :rtype: bool
         """
         return dereference(self.get_ptr()) == dereference(other.get_ptr())
     
     def __getstate__(self):
         """Pickle the SimplexTree data structure as a Python Byte Array
```

Comment:
Nit: does it pickle, or does it serialize for use with pickle?

Reply:
I fixed it and rephrased it on [d7d2049](https://github.com/GUDHI/gudhi-devel/pull/871/commits/d7d2049ca8e8b6e08ee5c979c7d132b8dbee9b7b)